### PR TITLE
minimega/bridge: fix capture

### LINKED ISSUE #545: Pcap capture 0 bytes
minimega  2016-05-08 works
minimega  2016-05-26 fails

Did a fresh install of latest same issue, something has changed.....

### PR DESCRIPTION
**Pull request by jcrussell**
Created a tuntap rather than letting ovs create the interface for us.
Capture now seems to work. Fixes #545.

### CODE REVIEW

**Comment by djfritz** [comment]
Why does this fix the issue? Was this just a race between invoking ovs creating a host tap and handing it to capture api for things?

File: src/bridge/mirror.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ func (b *Bridge) CreateMirror() (string, error) {
 	}
 
 	// get a host tap on VLAN 0
-	tap, err := b.createTap("", 0, true)
+	tap, err := b.createHostTap("", 0)
```
Comment (djfritz) [inline src/bridge/mirror.go]: Isn't this still having OVS create the tap via addTap()?

**Comment by jcrussell** [comment]
The OVS `add-port` command generated by minimega for host taps includes `set Interface <name> type=internal`. Currently, that command will automatically create the tap for us (and remove it when we `del-port`). When I created the tuntap first using `ip` and then tried to add it via `add-port`, the tuntap won't see any traffic.

There's probably something going on with OVS that I don't fully understand, perhaps there's an easier workaround? I'm copying the behavior that we had before the bridge refactoring.

**Comment by djfritz** [comment]
Woop, I saw this a "create a tuntap..." instead of "created a tuntap..." this makes sense now. LGTM once the merge conflict is fixed.

**Comment by djfritz** [comment]
LGTM

**Pull request merged**